backend/db/refresh_dataset_group_tables.py

```diff
@@ -109,6 +114,14 @@ def refresh_dataset_group_tables(
             # DB Counts
             print('Updating database counts. This could take a while...')
             counts_update(f'DB refresh: {",".join(dataset_group)}', schema)
+
+        # Vocab refresh only
+        # todo: ideally this notification would happen differently. More comments near bottom of refresh_voc.yml
+        if group_name == 'vocab' and len(dataset_group) == 1:  # Will be the case for GH actions
+            raise NotAnError('Notification: Restart deployment backend.\nThis is not an error. It is only being raised '
```

Comment:
## Vocab refresh: notification on success
- Now errors when completes successfully so that it will notify that a restart of the server is necessary to refresh networkx graph.